pages/ourteam.js

```diff
@@ -0,0 +1,180 @@
+          />
+          <header>
+            <NavbarBootstrap
+                isAuthenticated={false}
+            />
+            <div className="head">
+              <h1>Our Team</h1>
+              <div>
+                <p>
+                  We are a team of six University of Victoria Engineering students who formed
+                  Brakes Supreme as part of our capstone project course: ECE/SENG 499.<br/>
+                  Our group was formed in May 2019.
+                </p>
+              </div>
+            </div>
+          </header>
+
+          <main>
+            <div>
+              <div className="us">
+                <div className="us-one">
+                  <p>
+                    <Image
+                      width={140}
+                      height={140}
+                      src={require('../img/eli.jpg')}  />
+                      <p className="us-title">Eli</p>
+                      <p>Software/Hardware</p>
+                  </p>
+                </div>
+                <div className="us-two">
+                  <p className="us-picture">
+                    <Image
+                      width={140}
+                      height={140}
+                      src={require('../img/arshi.jpg')}  />
+                    <p className="us-title">Arshi</p>
+                    <p>Software</p>
+                  </p>
+                </div>
+                <div className="us-three">
+                  <p className="us-picture">
+                    <Image
+                      width={140}
+                      height={140}
+                      src={require('../img/shreyas.png')}  />
+                    <p className="us-title">Shreyas</p>
+                    <p>Software</p>
+                  </p>
+                </div>
+              </div>
+              <div className="us">
+                <div className="us-one">
+                  <p className="us-picture">
+                    <Image
+                      width={140}
+                      height={140}
+                      src={require('../img/rickus.jpg')}  />
+                    <p className="us-title">Rickus</p>
+                    <p>Hardware</p>
+                  </p>
+                </div>
+                <div className="us-two">
+                  <p className="us-picture">
+                    <Image
+                      width={140}
+                      height={140}
+                      src={require('../img/sajan.png')}  />
+                    <p className="us-title">Sajan</p>
+                    <p>Hardware</p>
+                  </p>
+                </div>
+                <div className="us-three">
+                  <p className="us-picture">
+                    <Image
+                      width={140}
+                      height={140}
+                      src={require('../img/adam.jpg')}  />
+                    <p className="us-title">Adam</p>
+                    <p>Software/Hardware</p>
```

Comment:
Maybe we can give more descriptive titles to everyone, like `lead software architect` or `embedded systems engineer`
what do you think @Eli-CB ?

Reply:
Agreed. The titles were temporary. I'll think of something